README.md

```diff
@@ -32,10 +32,14 @@ See full example [here](https://github.com/web-eid/web-eid-spring-boot-example).
 
 See below for alternative installation options.
 
-### 2. Add authentication support
+### 2. Activate CSRF protection
+
+Web eID requires CSRF protection. Enable and configure CSRF protection in your web application and include the CSRF token in the POST requests in the code examples below.
```

Comment:
Even though CSRF is highly recommended for security reasons, the Web-eID library and extension do not *require* CSRF to function.

```suggestion
Enable and configure CSRF protection in your web application and include the CSRF token in the POST requests in the code examples below.
```

Reply:
Thanks, right you are. Can we emphasize the importance of CSRF protection with the following sentence:

> _To use Web eID securely, CSRF protection must be enabled. Enable and configure ..._ (continues as before)